docs/manual-install.md

```diff
@@ -0,0 +1,149 @@
+      namespace: kube-system
+      labels:
+        tier: control-plane
+        component: azure-kms-provider
+    spec:
+      priorityClassName: system-node-critical
+      hostNetwork: true
+      containers:
+        - name: azure-kms-provider
+          image: mcr.microsoft.com/oss/azure/kms/keyvault:v0.0.11
+          imagePullPolicy: IfNotPresent
+          args:
+            - --keyvault-name=${KV_NAME}                            # [REQUIRED] Name of the keyvault
+            - --key-name=${KEY_NAME}                                # [REQUIRED] Name of the keyvault key used for encrypt/decrypt
+            - --key-version=${KEY_VERSION}                          # [REQUIRED] Version of the key to use
+            - -v=1
+          ports:
+            - containerPort: 8787
+              protocol: TCP
+          livenessProbe:
+            httpGet:
+              path: /healthz
+              port: 8787
+            failureThreshold: 2
+            periodSeconds: 10
+          resources:
+            requests:
+              cpu: 100m
+              memory: 128Mi
+            limits:
+              cpu: 4
+              memory: 2Gi
+          volumeMounts:
+            - name: etc-kubernetes
+              mountPath: /etc/kubernetes
+            - name: etc-ssl
+              mountPath: /etc/ssl
+              readOnly: true
+            - name: sock
+              mountPath: /opt
+      volumes:
+        - name: etc-kubernetes
+          hostPath:
+            path: /etc/kubernetes
+        - name: etc-ssl
+          hostPath:
+            path: /etc/ssl
+        - name: sock
+          hostPath:
+            path: /opt
+    ```
+
+    View logs from the kms pod:
+
+    ```bash
+    kubectl logs -l component=azure-kms-provider -n kube-system
+
+    I0219 17:35:33.608840       1 main.go:60] "Starting KeyManagementServiceServer service" version="v0.0.11" buildDate="2021-02-19-17:33"
+    I0219 17:35:33.609090       1 azure_config.go:27] populating AzureConfig from /etc/kubernetes/azure.json
+    I0219 17:35:33.609420       1 auth.go:66] "azure: using client_id+client_secret to retrieve access token" clientID="9a7a##### REDACTED #####bb26" clientSecret="23T.##### REDACTED #####vw-r"
+    I0219 17:35:33.609568       1 keyvault.go:66] "using kms key for encrypt/decrypt" vaultName="k8skmskv" keyName="key1" keyVersion="5cdf48ea6bb9456ebf637e1130b7751a"
+    I0219 17:35:33.609897       1 main.go:86] Listening for connections on address: /opt/azurekms.socket
+    ...
+    ```
+
+2. Create encryption configuration
+
+    Create a new encryption configuration file `/etc/kubernetes/manifests/encryptionconfig.yaml` using the appropriate properties for the `kms` provider:
+
+    ```yaml
+    kind: EncryptionConfiguration
+    apiVersion: apiserver.config.k8s.io/v1
+    resources:
+      - resources:
+          - secrets
+        providers:
+          - kms:
+              name: azurekmsprovider
+              endpoint: unix:///opt/azurekms.socket
```

Comment:
maybe add a comment for how this endpoint needs to match the one created by the plugin which is configurable